boot/jaxrs/src/main/java/org/commonjava/indy/boot/jaxrs/RestConfig.java

```diff
@@ -83,12 +83,15 @@ public void configureBuilder( final Undertow.Builder builder ) throws IOExceptio
         Xnio xnio = Xnio.getInstance( Undertow.class.getClassLoader() );
         final OptionMap.Builder omBuilder = OptionMap.builder()
                                                      .set( Options.WORKER_IO_THREADS, ioThreads )
-                                                     .set( Options.CONNECTION_HIGH_WATER, 1000000 )
+                                                     .set( Options.CONNECTION_HIGH_WATER, 2000000 )
                                                      .set( Options.CONNECTION_LOW_WATER, 1000000 )
                                                      .set( Options.TCP_NODELAY, true )
                                                      .set( Options.CORK, true )
                                                      .set( Options.WORKER_TASK_CORE_THREADS, workerThreads )
-                                                     .set( Options.WORKER_TASK_MAX_THREADS, workerThreads );
+                                                     .set( Options.WORKER_TASK_MAX_THREADS, workerThreads )
+                                                    .set(Options.KEEP_ALIVE, false)
```

Comment:
I'm really wondering if this is the magic behind this PR...have you tried with just these option changes in place to see?

Reply:
really i'm suppressed to see that it is with better performance for keep-alive to be false and not true. I have check it with false and true and difference is almost 20% without keep-alive
Check were done with apache tools with 100 000 request from which 100 concurrent.

Reply:
Ha , it seems that keep-alive option disabled is common practice on undertow servers where connections are dropped by server for some unexplain reason : https://github.com/reactor/reactor-netty/issues/388